### PR TITLE
Add required packages to setup.py

### PR DESCRIPTION
**Pull request by kdious**
Add packages required in core.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
     url='https://github.com/kingthomasc/python-purify',
     download_url='https://github.com/kingthomasc/python-purify/tarball/{ver}'.format(
         ver=version_num),
+    install_requires=['abc', 'six', 'xml'],
```
Comment (kdious) [inline setup.py]: Let me know your thoughts on this.  I ran into an issue when using a brand new ec2 image - `six` was missing and I had to add it manually.  Not sure if `abc` and `xml` are included as part of every python distribution.

**Comment by kdious** [comment]
👓 @kingthomasc

File: setup.py (hunk 1 above)
Comment (kd2718) [reply setup.py]: `abc` and `xml` are in the standard library. please just include `six` here.

**Comment by kd2718** [comment]
Good catch. Please see my comment and bump the version number up to `2.0.1` in the setup file.

I think after this I am going to try to setup proper tests and traviscli :smiley:

**Comment by kdious** [comment]
Bumped up the package.  I did not see any other comment.

**Comment by kdious** [comment]
@kingthomasc

**Review by kd2718 — APPROVED** [review]

**Pull request merged**